docs/deployment/heroku.md

```diff
@@ -8,6 +8,27 @@ The only thing we'll need for this guide is a working Phoenix application. For t
 
 Our main goal for this guide is to get a Phoenix application running on Heroku.
 
+### Limitations
+
+Heroku is a great platform overall, but for Elixir apps, it comes with a number of limitations you should be aware of. 
```

Comment:
It is great that are starting to see solutions that leverages all Elixir has to offer :D I am afraid though if you just drop Gigaelixir, it may do more harm than good, because how will the users even get started?

So what if, when suggesting alternatives, we link to a Gigaelixir guide? Could that be hosted on Gigaelixir? Let's also mention the distillery guide, provided by Phoenix, as an alternative.